src/pages/multipleProduct.tsx

```diff
@@ -0,0 +1,100 @@
+import { NextPageContext } from 'next';
+import { parseCookies } from 'nookies';
+import Layout from '../layout/Layout';
+import { OPERATOR_COOKIE, NUMBER_OF_PRODUCTS_COOKIE, MULTIPLE_PRODUCT_COOKIE } from '../constants';
+import ProductRow from '../components/ProductRow';
+import { ErrorInfo } from '../types';
+import ErrorSummary from '../components/ErrorSummary';
+import { MultiProduct } from './api/multipleProduct';
+
+const title = 'Multiple Product - Fares data build tool';
+const description = 'Multiple Product page of the Fares data build tool';
+
+export interface MultipleProductProps {
+    numberOfProductsToDisplay: string;
+    nameOfOperator: string;
+    errors?: ErrorInfo[];
+    userInput: MultiProduct[];
+}
+
+const MultipleProduct = ({
+    numberOfProductsToDisplay,
+    nameOfOperator,
+    errors = [],
+    userInput = [],
+}: MultipleProductProps): ReactElement => {
+    return (
+        <Layout title={title} description={description}>
+            <main className="govuk-main-wrapper app-main-class" id="main-content" role="main">
+                <form action="/api/multipleProduct" method="post">
+                    <ErrorSummary errors={errors} />
+                    <div className="govuk-form-group">
+                        <fieldset className="govuk-fieldset" aria-describedby="period-product-page-heading">
+                            <legend className="govuk-fieldset__legend govuk-fieldset__legend--xl">
+                                <h1 className="govuk-fieldset__heading" id="period-product-page-heading">
+                                    Enter your product details
+                                </h1>
+                            </legend>
+                            <span className="govuk-hint" id="service-operator-hint">
+                                {nameOfOperator} - {numberOfProductsToDisplay} Products
+                            </span>
+                        </fieldset>
+                        <div className="govuk-inset-text">For example, Super Saver ticket - £4.95 - 2</div>
+                    </div>
+                    <div className="govuk-grid-row">
+                        <ProductRow
+                            numberOfProductsToDisplay={numberOfProductsToDisplay}
+                            errors={errors}
+                            userInput={userInput}
+                        />
+                    </div>
+                </form>
+            </main>
+        </Layout>
+    );
+};
+
+export const getServerSideProps = (ctx: NextPageContext): { props: MultipleProductProps } => {
+    const cookies = parseCookies(ctx);
+
+    const baseOperatorCookie = cookies[OPERATOR_COOKIE];
+    const baseNumberOfProductsCookie = cookies[NUMBER_OF_PRODUCTS_COOKIE];
+
+    if (!baseOperatorCookie || !baseNumberOfProductsCookie) {
+        throw new Error('Necessary cookies not found to show multiple products page');
+    }
+
+    const operatorCookie = unescape(decodeURI(baseOperatorCookie));
```

Comment:
Since we're using this decode, unescape, parse pattern quite a lot, I'd say it would probably be worth making a helper function out of it

Reply:
Added one helper method for server side and one for client side. Might make this PR a little larger though...